docs/contributing/code-style.md

```diff
@@ -212,7 +212,7 @@ we should still avoid extremely long lines. A general guideline is:
 refactor stuff to get it under 85 characters, unless that makes the
 code a lot uglier, in which case it's fine to go up to 120 or so.
 
-### JavaScript
+### JavaScript and TypeScript
```

Comment:
This will presumably need to change to recommending arrow functions in TS, right?